Node/Task-3/src/controllers/users.controller.js

```diff
@@ -0,0 +1,34 @@
+const jwt = require('jsonwebtoken');
+
+const usersService = require("../services/users.service")
+
+/* 
+get list of users
+ */
+function getUsers(req, res, next) {
```

Comment:
you can use it as 
``` javascript
export async function getUsers(req, res, next) {
...
}
```

Reply:
Using `await usersService.getUsers()` inside `try catch` block?